frontend/web/components/tags/Tag.tsx

```diff
@@ -47,6 +47,7 @@ const Tag: FC<TagType> = ({
 
   return (
     <Tooltip
+      htmlEncode
```

Comment:
we'd probably want to make the property naming of this a bit more clear.

I think plainText might be good since in this case you're wanting to make the tooltip content text rather than html.

Reply:
Sure, that was my first option but then I thought it was not descriptive enough :). Done

Reply:
BTW, also fixed the component filename from toolip to tooltip.